cmd/kubectl-directpv/install.go

```diff
@@ -185,7 +185,9 @@ func installMain(ctx context.Context) {
 		OutputFormat:     outputFormat,
 		Declarative:      declarativeFlag,
 		Openshift:        openshiftFlag,
-		AuditWriter:      file,
+	}
+	if auditFile != nil {
+		args.AuditWriter = auditFile
```

Comment:
Just do
```diff
diff --git a/cmd/kubectl-directpv/install.go b/cmd/kubectl-directpv/install.go
index 3e2887ba..3e72cc3a 100644
--- a/cmd/kubectl-directpv/install.go
+++ b/cmd/kubectl-directpv/install.go
@@ -185,7 +185,9 @@ func installMain(ctx context.Context) {
 		OutputFormat:     outputFormat,
 		Declarative:      declarativeFlag,
 		Openshift:        openshiftFlag,
-		AuditWriter:      file,
+	}
+	if file != nil {
+		args.AuditWriter = file
 	}
 
 	var failed bool
```

Reply:
just renamed the file -> auditFile as it was confusing

Reply:
Make a small diff for the regression. You could add a comment about why we check non-nil and assign